WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductVariationFormViewModel.swift

```diff
@@ -93,12 +93,16 @@ final class ProductVariationFormViewModel: ProductFormViewModelProtocol {
 //
 extension ProductVariationFormViewModel {
     func canEditProductSettings() -> Bool {
-        return false
+        false
     }
 
     func canViewProductInStore() -> Bool {
         // no-op
```

Comment:
Should we remove this comment?

Reply:
yea makes sense! updated in https://github.com/woocommerce/woocommerce-ios/pull/2846/commits/19396ff0470831b2f357f600d2619d1f8373dfe5